src/Compilers/Core/CodeAnalysisTest/FileSystem/PathUtilitiesTests.cs

```diff
@@ -383,5 +383,13 @@ public void CombinePaths_DifferentFromPathCombine(string expected, string path1,
         {
             Assert.Equal(expected, PathUtilities.CombinePaths(path1, path2));
         }
+
+        [ConditionalFact(typeof(WindowsOnly)), WorkItem(51602, @"https://github.com/dotnet/roslyn/issues/51602")]
+        public void GetRelativePath_IndexOutOfRangeException()
+        {
+            var expected = "";
+            var result = PathUtilities.GetRelativePath(@"C:\A\B\", @"C:\A\B");
+            Assert.Equal(expected, result);
```

Comment:
instead of making this windows only, can we instead just build the string using the platform sepaarator?  or, barring that, have some tests for non-windows?